js/src/structured-data-blocks/how-to/utils/buildDurationString.js

```diff
@@ -0,0 +1,52 @@
+/* External dependencies */
+import { __ } from "@wordpress/i18n";
+import moment from "moment";
+import momentDurationFormatSetup from "moment-duration-format";
+
+momentDurationFormatSetup( moment );
+
+/**
+ * Formats the durations into a string.
+ *
+ * @param {Object} durations         The duration values.
+ * @param {string} durations.days    Number of days.
+ * @param {string} durations.hours   Number of hours.
+ * @param {string} durations.minutes Number of minutes.
+ *
+ * @returns {string} Formatted duration.
+ */
+export default function buildDurationString( durations ) {
+	const durationDays = durations.days ? parseInt( durations.days, 10 ) : 0;
+	const durationHours = durations.hours ? parseInt( durations.hours, 10 ) : 0;
+	const durationMinutes = durations.minutes ? parseInt( durations.minutes, 10 ) : 0;
+
+	const elements = [];
+	if ( durationDays !== 0 ) {
+		elements.push( `d [${ __( "days", "wordpress-seo" ) }]` );
+	}
+	if ( durationHours !== 0 ) {
+		elements.push( `h [${ __( "hours", "wordpress-seo" ) }]` );
+	}
+	if ( durationMinutes !== 0 ) {
+		elements.push( `m [${ __( "minutes", "wordpress-seo" ) }]` );
+	}
```

Comment:
Question: while it seems the singular and plural forms are handled properly in English:

<img width="486" alt="screen shot 2018-08-28 at 18 58 29" src="https://user-images.githubusercontent.com/1682452/44738632-cb573680-aaf5-11e8-8094-67121786bed4.png">

hos is this going to work with other languages? I see only the plural forms are translatable, while for example in Italian, both singular and plural should be translatable:
1 giorno, 2 giorni
1 ora, 2 ore
1 minuto, 2 minuti

Also, as a general best practice, building a translatable string putting together various small pieces of text containing variables is far from ideal. I understand there are no great options in this case. However, in many languages, a date/time indication can have different forms. For example, we shouldn't assume a comma as separator is correct in all languages, as well the word `and` before the minutes. @SergeyBiryukov has a lot of experience with translations-related stuff, I'd recommend to take advantage of his expertise.